eva_cttv_pipeline/evidence_strings.py

```diff
@@ -147,6 +167,37 @@ def __init__(self):
         # CTTVEvidenceString.__init__(self, a_dictionary)
         super().__init__(a_dictionary)
```

Comment:
This call could have more arguments which are initialised the same way in both child classes (like `ensembl_gene_id` or `clinvarRecord`).